### ISSUE TITLE
Struct error on format input type

### ISSUE THREAD

**Issue by PackeTsar** [msoulier/tftpy#78]
Running Py2.7.5 on CentOS7. After starting up the server and mounting to root (`/`), when client makes a request, server throws the below exception.

```
[root@CENT7 tftpy]# tftpy_server.py -r "/"
INFO:tftpy:Server requested on ip 0.0.0.0, port 69
INFO:tftpy:Starting receive loop...
Traceback (most recent call last):
  File "/usr/bin/tftpy_server.py", line 59, in <module>
    main()
  File "/usr/bin/tftpy_server.py", line 51, in main
    server.listen(options.ip, options.port)
  File "/usr/lib/python2.7/site-packages/tftpy/TftpServer.py", line 166, in listen
    self.sessions[key].start(buffer)
  File "/usr/lib/python2.7/site-packages/tftpy/TftpContexts.py", line 234, in start
    pkt = self.factory.parse(buffer)
  File "/usr/lib/python2.7/site-packages/tftpy/TftpPacketFactory.py", line 28, in parse
    (opcode,) = struct.unpack("!H", buffer[:2])
TypeError: Struct() argument 1 must be string, not unicode
```

**Comment by PackeTsar** [comment]
Forking and fixing. Will submit a PR soon.

**Comment by msoulier** [comment]
Hmm. Why is this not happening in the unit tests...?

**Comment by msoulier** [comment]
Should be fixed in 45694f2902af7472fea903d3c6e71a637bc10e18.
